Common/src/Pollster/TaskHandler.cs

```diff
@@ -131,8 +131,17 @@ public async ValueTask DisposeAsync()
     onDispose_.Invoke();
     logger_.LogDebug("MessageHandler status is {status}",
                      messageHandler_.Status);
-    await messageHandler_.DisposeAsync()
-                         .ConfigureAwait(false);
+    try
+    {
+      await messageHandler_.DisposeAsync()
+                           .ConfigureAwait(false);
+    }
+    catch (Exception e)
+    {
+      logger_.LogWarning(e,
+                         "Error while disposing message handler");
```

Comment:
```suggestion
      logger_.LogWarning(e,
                         "Error while disposing message handler {MessageHandler}. It will appear duplicated",
                         messageHandler_.MessageId);
```